### PR TITLE
feat(setRequestList): add new filter and update filters UX

### PR DESCRIPTION
**Pull request by wescopeland**
This PR revises the set request list page filters UX via a Blade migration and adds an additional filter to see all, only claimed, or only unclaimed games.

ref: https://discord.com/channels/310192285306454017/1105656936998903838

---

https://github.com/RetroAchievements/RAWeb/assets/3984985/0ec81367-2cf1-44a4-b24f-d4af6d5276fe

### CODE REVIEW

**Comment by wescopeland** [comment]
I have revised the UX for this PR and migrated the filters panel to Blade.

File: resources/views/community/components/request-list/meta-panel.blade.php [hunk 1]
```diff
@@ -0,0 +1,69 @@
+<script>
+/**
+ * Updates a query parameter in the current URL and navigates to the new URL.
+ *
+ * @param {string} paramName - The name of the query parameter to update.
+ * @param {string} newQueryParamValue - The new value for the query parameter.
+ */
+ function updateUrlParameter(paramName, newQueryParamValue) {
```
Comment (wescopeland) [inline resources/views/community/components/request-list/meta-panel.blade.php:8]: Unfortunately, this JS is all untestable. Node.js and JSDOM support only an extremely thin implementation of `URLSearchParams`.

File: resources/views/community/components/request-list/meta-console-filter.blade.php [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        <option selected>All systems</option>
+    @else
+        <option value="-1">All systems</option>
+    @endif
+
+    @foreach ($consoles as $console)
+        @if ($selectedConsoleId == $console['ID'])
+            <option selected>{{ e($console->Name) }}</option>
+        @else
+            <option value="{{ $console['ID'] }}">{{ e($console->Name) }}</option>
```
Comment (Jamiras) [inline resources/views/community/components/request-list/meta-console-filter.blade.php:19]: This seems to be doing some sort of escaping:
image

Comment (wescopeland) [reply resources/views/community/components/request-list/meta-console-filter.blade.php]: Great catch, it appears this is totally unnecessary and Blade automatically handles what `sanitize_outputs()` does in legacy. Addressed in https://github.com/RetroAchievements/RAWeb/pull/1574/commits/d083e4d665b470fe1c5ac58f6a9d32cde2b9611f.

**Comment by wescopeland** [comment]
@luchaos @Jamiras This branch has been updated to resolve all outstanding conflicts with V3.

**Review by Jamiras — APPROVED** [review]

**Pull request merged**